aocd/examples.py

```diff
@@ -118,12 +125,12 @@ class Example(NamedTuple):
     """
 
     input_data: str
-    answer_a: str = None
-    answer_b: str = None
-    extra: str = None
+    answer_a: str | None = None
+    answer_b: str | None = None
+    extra: str | None = None
 
     @property
-    def answers(self):
+    def answers(self) -> tuple[str | None, str | None]:
```

Comment:
What is the code-path where these can be `None`?  IIRC we raise exception for unavailable answers, and day 25 part b answer is always the empty string.

Reply:
I think you were thinking of `Puzzle.answers` here, which can only return `tuple[str, str]` and will raise an exception if that's not possible. But anyone can create an `Example` instance with one or other answer ommitted.